nisystemlink/clients/core/_internal/_classproperty_support.py

```diff
@@ -31,8 +33,8 @@ class ClasspropertySupport_(meta):  # type: ignore
         )
 
     @classmethod
-    def classproperty(cls, f: Callable[[Any], Any]) -> property:
+    def classproperty(cls, f: Callable[[Any], _T]) -> _T:
```

Comment:
Note: Something updated such that callers of a method decorated with `@classproperty` was seeing the return type as `property` instead of the intended type. This changes the typing to match the return type of the method being wrapped.